asyn/asynDriver/asynManager.c

```diff
@@ -308,6 +310,7 @@ static asynStatus exceptionDisconnect(asynUser *pasynUser);
 static asynStatus interposeInterface(const char *portName, int addr,
     asynInterface *pasynInterface,asynInterface **ppPrev);
 static asynStatus enable(asynUser *pasynUser,int yesNo);
+static asynStatus shutdown(asynUser *pasynUser);
```

Comment:
Not a functional issue, but maybe better to pick a function name which isn't shared with a syscall?